src/main/features/response/guards/oweSomePaidNoneResponseRequiredGuard.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+
+import { ResponseType } from 'response/form/models/responseType'
+import { Paths } from 'response/paths'
+import User from 'idam/user'
+
+export default class OweSomePaidNoneResponseRequiredGuard {
```

Comment:
That class seems not to be used at all. What is the point of having guard / a class which is not used?